snuba/datasets/storages/errors.py

```diff
@@ -85,6 +103,12 @@ def errors_migrations(
         ("user_email", Nullable(String())),
         ("sdk_name", LowCardinality(Nullable(String()))),
         ("sdk_version", LowCardinality(Nullable(String()))),
+        (
+            "url",
+            Materialized(Nullable(String()), "tags.value[indexOf(tags.key, 'url')]"),
+        ),
```

Comment:
Wait, I checked what is in the events table. The url column is a promoted promoted tag, as such it should only be accessed as `tags[url]`, it is never filled in manually from an interface. 
So if we are reproducing the behavior of events, this column should not be needed at all here.

Reply:
Don't I still need to add the column to the table, so that the tag can be promoted into it?

Reply:
I don't think we need to promote that tag until there is a performance reason for doing it. Different tables don't have the same set of promoted tags. We generally start without promoting, and add promoting when we have evidence it is needed.